### ISSUE TITLE
Lost welcome message

### ISSUE THREAD

**Issue by 1-alex98** [FAForever/server#562]
From zulip

Alexander von Trostorff aka axel12(00)? 11:30 AM
Hi @Askaholic @MazorNoob (not sure if u do stuff on the server) @UmbraSolis / cleborys and whoever can have a look.

There is a bunch of users reporting getting stuck in the login screen, basically since 3-4 days or so (not sure exactly). Since we did not change the client coded related to that lately I actually suspect the server to be behind this. I also consider it being a client issue. I took precautions and increased login and added additional error handling in the places related(gonna come soon v1.1.6). Here is an exaple forum post https://forums.faforever.com/viewtopic.php?f=3&t=18996&p=183021#p183021 (got a lot more on discord tec help and on zulip). So from the log we can see he receives the "session" message and then is missing a received "welcome" message. The question is where is the "welcome" message and where could it have been gone.

I know you were working on the async stuff and it lost messages as it seems. Maybe u added async stuff with login and now that is loosing messages. Well maybe that's bullshit just an idea.

  11:32 AM
I am gonna open a github issue with the same text

**Comment by Askaholic** [comment]
From the server log, the user in question seems to have been kicked out by a second connection:
```
WARNING  Apr 02  01:29:56 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  01:34:36 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  01:40:10 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  01:45:29 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  01:49:24 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  02:01:09 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  02:14:34 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  03:16:11 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  03:31:23 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  03:33:33 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
WARNING  Apr 02  04:35:12 LobbyConnection                Client Defiant dropped. You have been signed out because you signed in elsewhere.
```
The times here are in UTC, maybe we can verify with the user if these timestamps correspond to the times when they were having issues logging in.

**Comment by faf-nsw** [comment]
To Askaholic's post, I believe the times are correct.   Next day I was able to login with no change made on my client or account.

NSW/Def

**Comment by axel1200** [comment]
Ok so u say if u are logged in else where that is what happens??? Well we should be able to reproduce this then right?

**Comment by Askaholic** [comment]
Anyone know if the infinite login still happens? We deployed a possible fix for it (#670) a while ago and I haven’t seen any complaints about this issue in discord or on the forums lately.

**Comment by Askaholic** [comment]
I am going to assume this issue has been fixed.
